improver_tests/nbhood/nbhood/test_BaseNeighbourhoodProcessing.py

```diff
@@ -241,39 +223,36 @@ def setUp(self):
         data[7, 7] = 0
         self.cube = set_up_variable_cube(data, spatial_grid="equalarea",)
 
-        time_points = [
-            datetime(2017, 11, 10, 2),
-            datetime(2017, 11, 10, 3),
-            datetime(2017, 11, 10, 4),
-        ]
-        self.multi_time_cube = add_coordinate(
-            self.cube, coord_points=time_points, coord_name="time", is_datetime="true",
-        )
-
-    def test_basic(self):
-        """Test that the plugin returns an iris.cube.Cube."""
-        neighbourhood_method = Neighbourhood("circular")
-        result = NBHood(neighbourhood_method, self.RADIUS)(self.cube)
-        self.assertIsInstance(result, Cube)
-
-    def test_neighbourhood_method_does_not_exist(self):
-        """
-        Test that desired error message is raised, if the neighbourhood method
-        does not exist.
-        """
-        neighbourhood_method = "nonsense"
-        radii = 10000
-        msg = "is not valid as a neighbourhood_method"
-        with self.assertRaisesRegex(ValueError, msg):
-            NBHood(neighbourhood_method, radii)(self.cube)
-
     def test_single_point_nan(self):
         """Test behaviour for a single NaN grid cell."""
         self.cube.data[6][7] = np.NAN
         msg = "NaN detected in input cube data"
         with self.assertRaisesRegex(ValueError, msg):
-            neighbourhood_method = Neighbourhood
-            NBHood(neighbourhood_method, self.RADIUS)(self.cube)
+            NBHood(self.RADIUS)(self.cube)
+
+    def test_correct_radii_set(self):
+        """Test that the correct neighbourhood radius is set when interpolation
+         is required"""
+
+        radii = [5600, 9500]
+        lead_times = [3, 5]
+        expected_radius = 7550
+        plugin = NBHood(radii, lead_times)
+        plugin(self.cube)
+        self.assertEqual(plugin.radius, expected_radius)
+
+
+class Test_process_old(IrisTest):
```

Comment:
The class above contains the process method unit tests I think we now need for the simplified base class. 

This class Test_process_old contains unit tests that tested the original more complex base class through the interaction between the base class and the neighbourhood processing square and circular classes. I've left these unit tests here for now, to show I haven't broken them, but they will be moved and consolidated with the other neighbourhood processing unit tests in  a later PR.